tools/test/test_upload_test_stats.py

```diff
@@ -0,0 +1,21 @@
+import unittest
+import os
+
+IN_CI = os.environ.get("CI")
+
+from tools.stats.upload_test_stats import get_test_cases
+
+
+class TestUploadTestStats(unittest.TestCase):
```

Comment:
So is the advice here always to run this locally before submitting a pr?

Reply:
yeah, I figured it was fine because we are the only people who really change things related to this test. If it becomes a problem in the future, we *could* just run it probably.